docs/02_getting-started/03_smart-contract-development/11_create-tic-tac-toe-contract.md

```diff
@@ -0,0 +1,709 @@
+---
+content_title: "2.10 Creating a Tic-Tac-Toe Smart Contract"
+link_text: "2.10 Creating a Tic-Tac-Toe Smart Contract"
+---
+
+## Goal
+
+The following tutorial will guide you to build a sample Player vs Player game contract. We will apply the knowledge we acquired eariler and use the tic tac toe game to demonstrate.
```

Comment:
Since we have used the hyphenated phrase for **Tic-Tac-Toe** in the content title, we can keep it consistent for all instances appearing throughout the tutorial except for the .cpp and .hpp file names. 

So either we can keep it hyphenated or non hyphenated. I checked the internet and I see tic tac toe represented in both the forms - **Tic-Tac-Toe** and also **Tic Tac Toe.** I think I prefer the hyphenated version. You can make the final call.

Reply:
The hyphenated one is good. Could you update them?

Reply:
Okay let me do that.